### ISSUE TITLE
Validating based on extension during dragging is unreliable 

### ISSUE THREAD

**Issue by lukemelia** [tim-evans/ember-file-upload#127]
Unfortunately, PR #121 does not copy out the file name, which is needed to validate based on extension. When I set out to add this, I discovered that some browsers (Chrome, at least) do not make the file name available while dragging. Therefore, it appears to be impossible to reliably validate during drag based on file extensions. Additionally, since some files (e.g., .xls on my Mac) have no mime type set, it appears to be impossible to rely on mime type exclusively except for very standard file types. This leaves us in a tough spot.

**Comment by tim-evans** [comment]
Yeah, I believe this is a security issue, which makes it frustrating for us trying to make a better experience for our users. I'm open to suggestions for this, but I think the best approach will be to remove validation on drag & drop, since it's unreliable.

**Comment by lukemelia** [comment]
I tend to agree @tim-evans. The workaround I just implemented (we'll see how it holds up) is that if my `accept` prop includes non-standard mime types, I have a CP that indicates that validation is not reliable. So while dragging, my dropzone subclass has three potential states: valid, invalid, and "can't reliably say".

**Comment by tim-evans** [comment]
😂 Well, that sounds just about right.
